content/en/blog/2023/tyk-api-gateway/index.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Tyk API Gateway's Native OpenTelemetry Integration
+linkTitle: Tyk API Gateway's Native OpenTelemetry Integration
+date: 2023-10-24
+author: >-
+  [Sonja Chevre](https://github.com/SonjaChevre) (Tyk Technologies)
+draft: false
+---
+
+{{% alert color="primary" %}} _It is part of
+[OpenTelemetry's vision](http://localhost:1313/community/mission/#telemetry-should-be-built-in),
+that we believe that high-quality telemetry can be built in to the entire
+software stack. Therefore we are excited, that there are more and more
+[libraries, services and apps, that integrate OpenTelemetry](/ecosystem/integrations/)
+to provide built in observability to their users._
+
+_In this and future blog post, we will spot light these integrations to you! If
+you want your integration to be listed and to be highlighted in a blog post as
+well,
+[reach out](https://github.com/open-telemetry/opentelemetry.io/issues/new)!_
+{{% /alert %}}
+
```

Comment:
@SonjaChevre @austinlparker @cartersocha PTAL!

Reply:
looks good from my side!

Small editing suggestion for the first sentence:

instead of

> It is part of
[OpenTelemetry's vision](http://localhost:1313/community/mission/#telemetry-should-be-built-in),
that we believe that 

> As part of [OpenTelemetry's vision](http://localhost:1313/community/mission/#telemetry-should-be-built-in), we believe that

Reply:
```suggestion
{{% alert color="primary" %}}
As part of [OpenTelemetry's vision](http://localhost:1313/community/mission/#telemetry-should-be-built-in),
we believe that high-quality telemetry can be built in to the entire
software stack. Therefore we are excited, that there are more and more
[libraries, services and apps, that integrate OpenTelemetry](/ecosystem/integrations/)
to provide built in observability to their users._

_In this and future blog post, we will spot light these integrations to you! If
you want your integration to be listed and to be highlighted in a blog post as
well,
[reach out](https://github.com/open-telemetry/opentelemetry.io/issues/new)!_
{{% /alert %}}

```

Reply:
thanks @SonjaChevre, updated :-)

Reply:
"In this and future blog post(s), we will spot light these integrations." 
"to be listed and highlighted" (maybe my grammar is off here but seems repetitive as is
- minor nits

Reply:
@cartersocha @svrnm et al.: how about a **flashy badge** instead of a note? See the following proposal:

- #3485

Reply:
superseded by https://github.com/open-telemetry/opentelemetry.io/pull/3431#discussion_r1383549296